hubmap/frontend/src/ui-components/index.js

```diff
@@ -0,0 +1,9 @@
+/** *
```

Comment:
Again, is there a reason why `ui-components/index.js` should just be focussed on the browse page?

Reply:
removed